core/src/test/scala/com/databricks/labs/remorph/parsers/tsql/TSqlFunctionBuilderSpec.scala

```diff
@@ -18,6 +17,7 @@ class TSqlFunctionBuilderSpec extends AnyFlatSpec with Matchers with TableDriven
       ("functionName", "expectedArity"), // Header
 
       // TSql specific
+      (s"$$PARTITION", Some(FunctionDefinition.notConvertible(0))),
```

Comment:
```suggestion
      ("$PARTITION", Some(FunctionDefinition.notConvertible(0))),
```

Reply:
This does not seem to give the error, but given it gives it elsewhere, I thought it better to be consistent.